docs/src/traits.md

```diff
@@ -0,0 +1,18 @@
+# Traits
+
+```@docs
+Transform
+assertions
+Stateless
+Colwise
+colapply
+colrevert
+TableTransforms.colcache
```

Comment:
Maybe we need to export colcache as well?

Reply:
I think so. But I don't know if it's interesting to export these internal functions.

Reply:
Maybe we should unexport colapply and colrevert. What do you think?

Reply:
I think all these functions and types from the Trait section are part of the internal API. I don't think they need to be exported.

Reply:
That is a good point. Maybe a separate PR fixing the exports first before we can finish this one here about docs.